dlp/dlp_snippets/inspect_test.go

```diff
@@ -44,6 +44,33 @@ func TestInspectString(t *testing.T) {
 	}
 }
 
+func TestInspectStringWithCustomInfoTypes(t *testing.T) {
```

Comment:
If this stays as a single function, I'd prefer if there were a single test (can add fields to the tests struct).

Reply:
Done.